### PR TITLE
Add flexVolume to list and add anchor.

### PR DESCRIPTION
**Pull request by Zefool**

### CODE REVIEW

File: content/en/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -74,7 +74,8 @@ Kubernetes supports several types of Volumes:
    * [csi](#csi)
    * [downwardAPI](#downwardapi)
    * [emptyDir](#emptydir)
-   * [fc (fibre channel)](#fc)
+   * [fc (fibre channel)](#fc)flexVolume
```
Comment (tengqm) [inline content/en/docs/concepts/storage/volumes.md]: ?

Comment (Zefool) [reply content/en/docs/concepts/storage/volumes.md]: Huh. No idea what happened here.

Comment (Zefool) [reply content/en/docs/concepts/storage/volumes.md]: Should be fine now. Thanks for reviewing.

**Comment by Zefool** [comment]
@tengqm any updates here? Anything I need to do?

**Comment by tengqm** [comment]
@Zefool Thanks for the fix.
/lgtm
/approve

**Pull request merged**